{
	"id": "CVE-2020-26242",
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2020-26242",
				"title": "Denial of service in geth",
				"description": "Go Ethereum, or \"Geth\", is the official Golang implementation of the Ethereum protocol. In Geth before version 1.9.18, there is a Denial-of-service (crash) during block processing. This is fixed in 1.9.18.",
				"severity": [
					{
						"type": "cvss_v31",
						"source": "GitHub_M",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H",
							"base_score": 6.5,
							"base_severity": "MEDIUM",
							"temporal_score": 6.5,
							"temporal_severity": "MEDIUM",
							"environmental_score": 6.5,
							"environmental_severity": "MEDIUM"
						}
					}
				],
				"references": [
					{
						"source": "CVE",
						"url": "https://blog.ethereum.org/2020/11/12/geth_security_release/"
					},
					{
						"source": "CVE",
						"url": "https://github.com/ethereum/go-ethereum/security/advisories/GHSA-jm5c-rv3w-w83m"
					},
					{
						"source": "GitHub_M",
						"url": "https://blog.ethereum.org/2020/11/12/geth_security_release/"
					},
					{
						"source": "GitHub_M",
						"url": "https://github.com/ethereum/go-ethereum/security/advisories/GHSA-jm5c-rv3w-w83m"
					}
				],
				"modified": "2024-08-04T15:56:03.57Z"
			}
		}
	],
	"data_source": {
		"id": "mitre-v5",
		"raws": [
			"vuls-data-raw-mitre-v5/2020/CVE-2020-26242.json"
		]
	}
}
